package.json

```diff
@@ -10,8 +10,6 @@
     "rust:test": "cd src-tauri; cargo test",
     "ui:dev": "vite",
     "ui:build": "tsc && vite build",
-    "ui:lint": "eslint src/*.{ts,tsx}",
```

Comment:
Can we expose a command for devs to manually lint the project?

Reply:
Fixed